resolwe_bio/tools/cuffnorm_output_table_headers.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python2
+# pylint: disable=missing-docstring,invalid-name
+# XXX: Refactor to a comand line tool and remove pylint disable
+"""Rename cuffnorm output tables headers."""
+from __future__ import absolute_import, division, print_function
+
+import argparse
+
+parser = argparse.ArgumentParser(description="Rename cuffnorm output tables headers")
+parser.add_argument('sample_names', help="Sample names")
+args = parser.parse_args()
+
+sample_names=[]
+
+for sample_file in args.sample_names.split(","):
+    sample_names.append(sample_file.split(".")[0])
+
+
+working_array=[]
+with open('cds.count_table', 'r') as genes_table:
+    genes_table.next()
+    for line in genes_table:
+        working_array.append(line)
+
+with open('cds.count_table', 'w') as genes_table:
+    genes_table.write("tracking_id"+'\t')
+    for sample in sample_names:
+        genes_table.write("%s" %sample + '\t')
```

Comment:
Use `format` instead of `%`. Check [this link](https://pyformat.info/) for more info.